cumulus_library/template_sql/codeable_concept_denormalize.sql.jinja

```diff
@@ -21,7 +21,7 @@ CREATE TABLE {{ target_table }} AS (
         {%- endif %}
         {%- if filter_priority %}
         WHERE
-            u.codeable_concept.system = '{{ system }}'
+            u.codeable_concept.system LIKE '{{ system }}%'
```

Comment:
Oh thats.. interesting. As a developer passing a system around, I would be very surprised to see it used as a prefix rather than a black-box string. `http://example.org/animals/` being different from `http://example.org/animals/lions` for example. What's the use case for this - private urn namespaces? Is there maybe a way to opt-into the wildcard nature? But even then, I'm leery that we get a promise about all possible meanings / systems in the namespace?

Reply:
there's a lot of noise in the numeric ones specifically, and without the context, I couldn't determine what they corresponded to and how comprehensive they were, so rather than putting in 15 codes i tried to cheat it - i can flip this back and be more explicit.

Reply:
ok, per our conversation offline, i've updated this to the explicit IDs, and removed the installation-specific guids from the cerner IDs.

I created #180 to track the need to think about some optimizations as these lists get larger.